exec/java-exec/src/test/java/org/apache/drill/exec/physical/impl/limit/TestLimitWithExchanges.java

```diff
@@ -125,6 +125,15 @@ public void testLimitImpactExchange() throws Exception {
     }
   }
 
+  @Test
+  public void TestLimitAllOnParquet() throws Exception {
```

Comment:
Have you tried this new testcase without the change above? Will it successful or fail?